libs/iostability/top_stats_collection.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# !/usr/bin/python
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+"""Methods to collect top command stats from server"""
+
+import logging
+
+from commons.commands import KILL_CMD, K8S_APPLY_FILE
+from commons.constants import PROFILE_FILE_PATH, COLLECTION_SCRIPT_PATH, COLLECTION_FILE, \
+    PROFILE_FILE
+from commons.helpers.pods_helper import LogicalNode
+from multiprocessing import Process
+# check and set pytest logging level as Globals.LOG_LEVEL
+LOGGER = logging.getLogger(__name__)
+
+
+class TopStatsCollection:
+    """
+        This class contains common methods to collect stats of top cmd from k8s system
+    """
+    def __init__(self, cmn_cfg):
+        """
+        Initialize connection to Nodes or Pods.
+        :param cmn_cfg: Common config
+        """
+        self.cmn_cfg = cmn_cfg
+        self.nodes = cmn_cfg["nodes"]
+        self.master_node_list = list()
+        for node in self.nodes:
+            if node["node_type"].lower() == "master":
+                node_obj = LogicalNode(hostname=node["hostname"],
+                                       username=node["username"],
+                                       password=node["password"])
+                self.master_node_list.append(node_obj)
+
+    @staticmethod
+    def exe_cmd(node: LogicalNode, cmd: str):
+        """function to execute command collection
+        :param node: Object of Logical node
+        :param cmd: command to be executed on worker node
+        """
+        LOGGER.debug("executing procpath cmd for stat collection")
```

Comment:
if this is specifically for procpath maybe method name can be for procpath and cmd can be directly stated here?
If this is common method to execute any commnd then this logger needs chnages

Reply:
Changing logger statement